math-magicians/src/App.js

```diff
@@ -7,7 +7,7 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.js</code> and save to reload.
+          Hello world!
```

Comment:
* @Enoisong as i explained previously you should only have one folder housing all your react files currently you have `MATH-MAGICIANS` folder and inside it you then have another `math-magicians` which houses your react app, because of this i have an error when i try to run `npm start` , Kindly cut all the files inside `math-magician` folder ( eg public, src etc)  and paste it inside the `MATH-MAGICIANS` so that all your react app Files are all in one folder and not nested 
![image](https://github.com/Enoisong/math-magicians/assets/52497246/b9f5f994-a80b-4994-bfd0-30f32bfa7088)
![image](https://github.com/Enoisong/math-magicians/assets/52497246/60939a46-cb04-4913-9ef8-73f89660fbb8)